CopilotKit/packages/react-textarea/src/hooks/make-autosuggestions-function/use-make-standard-autosuggestions-function.tsx

```diff
@@ -52,14 +50,20 @@ export function useMakeStandardAutosuggestionFunction(
           },
         ];
 
-        const apiEndpoint = ChatlikeApiEndpoint.fromCopilotApiConfig(copilotApiConfig);
```

Comment:
there's an issue with the new implementation -
`fromCopilotApiConfig` integrates properties from the `copilotApiConfig` that we are now ignoring (.body, .headers)